handbook/product/README.md

```diff
@@ -393,7 +393,11 @@ Directly Responsible Individuals (DRI) engage in the ritual(s) below at the freq
 | Bug review (CX)                               | Weekly (Tuesday)         | The product manager meets with the engineering manager and a product quality engineer to ensure synchronization on bug prioritization and actions preceding CX estimation sessions.                                                                                                         | Mo Zhu            |
 | Churned bug review                            | Sprintly (Wednesday)     | The Head of Product meets with the group product managers to examine churned bugs and categorize them as either continue, needs prioritization, or de-prioritize.                                                                                                                                     | Mo Zhu            |
 | Churned bug clean-up | Sprintly | Following the bug de-prioritization meeting, Product Operations completes the churned bug clean-up, ensuring all necessary follow-up tasks are actioned to classify bugs as continue, needs prioritization, or de-prioritized. This may include relocating bug tickets, adjusting labels, communicating with stakeholders, writing documentation, and closing issues. | Isabell Reedy |
+Bug round-up | Sprintly (Tuesday) | After each sprint, Product Operations will compile a list of churned bugs, which will detail the issue number, bug specifics, bug age, and an area for product managers to suggest subsequent actions.| Isabell Reedy |
+Quality check | Daily | Every day, Product Operations will review the "designed" column on the drafting board to ensure all product action items are complete. | Isabell Reedy |
 
+### Product calendar 
+The product calendar is the go-to source for all current product rituals. As a centralized and shared resource, it keeps the product team aligned, promoting a steady flow of operations each sprint. Access it [here](https://calendar.google.com/calendar/embed?src=c_644c399c66ea0f1e2e85904b6c63a93ec7ff4aa8796868891bfeab7445bf3179%40group.calendar.google.com&ctz=America%2FNew_York).
```

Comment:
I think the issue template for onboarding new people should also include a reference to this calendar.

Reply:
@zhumo On it! I have added you as a reviewer to the onboarding issue PR.